### ISSUE TITLE
Integration using CocoaPods without `use_frameworks!`

### ISSUE THREAD

**Issue by skyline75489** [rs/SDWebImage#1700]
### New Issue Checklist
- [X] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
- [X] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
- [X] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none
### Issue Info

| Info | Value |
| --- | --- |
| Platform Name | ios |
| Platform Version | 7.0 |
| SDWebImage Version | 4.0.0-beta2 |
| Integration Method | cocoapods 1.0.1 |
| Xcode Version | Xcode 8 |
| Repro rate | all the time (100%) |
| Repro with our demo prj | no |
| Demo project link | - |
### Issue Description and Steps

I'm trying to integrate SDWebImage using CocoaPods without `use_frameworks!` but failed at

``` objectivec
#if COCOAPODS
@import FLAnimatedImage;
#else
#import "FLAnimatedImageView.h"
#endif
```

I searched the issue list and found #1579 . The issue was closed. However the error remains. Am I missing something here?

I came up with #1696 which should fix this problem.

**Comment by gujinku** [comment]
I am using cocoaPods 1.1.0 r2.  I integrated it into my project successfully. But not sure why you remove "use_frameworks!"

**Comment by skyline75489** [comment]
@gujinku Well, as a full Objective-C project, it's not necessary to use SDWebImage with "use_frameworks!".

**Comment by bpoplauschi** [comment]
I agree @skyline75489. We need to support both frameworks and static libs integrations. I will look into this issue soon.

**Comment by HuangLily** [comment]
When can SDWebImage support static libs?

**Comment by andrey-morozov** [comment]
a similar issue. I'm trying to integrate SDWebImage using CocoaPods without use_frameworks! but failed at:
 ld: framework not found SDWebImage for architecture arm64

1. Platform Version SDWebImage 3.8.2
2. IOS dep target 8.1
3. CocoaPods 1.1.1

**Comment by bpoplauschi** [comment]
Fixed by #1696
